app/views/claimToAdjustPoa/SuccessView.scala.html

```diff
@@ -0,0 +1,76 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.html.layouts.unifiedLayout
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import models.incomeSourceDetails.TaxYear
+@import views.html.components._
+@import _root_.implicits.ImplicitCurrencyFormatter._
+
+@this(
+        mainTemplate: unifiedLayout,
+        govukPanel: GovukPanel,
+        p: p,
+        link: link,
+)
+
+@(isAgent: Boolean, poaTaxYear: TaxYear, poaTotalAmount: BigDecimal)(implicit messages: Messages, request: RequestHeader, user: auth.MtdItUser[_])
+
+@getMessage(key: String, args: String*) = @{
+    messages(s"claimToAdjustPoa.success.$key", args: _*)
+}
+
+@taxYearSummaryUrl = @{
+    if(isAgent) {
+        controllers.routes.TaxYearSummaryController.renderTaxYearSummaryPage(poaTaxYear.endYear).url
+    } else {
+        controllers.routes.TaxYearSummaryController.renderTaxYearSummaryPage(poaTaxYear.endYear).url
+    }
+}
+
+@whatYouOweUrl = @{
+    if(isAgent) {
+        controllers.routes.WhatYouOweController.showAgent.url
+    } else {
+        controllers.routes.WhatYouOweController.show().url
+    }
+}
+
+@mainTemplate(pageTitle = getMessage("heading"),
+    isAgent = isAgent) {
+
+    @govukPanel(Panel(
+        title = Text(getMessage("h1")),
+        content = HtmlContent(getMessage("p1", poaTaxYear.startYear.toString, poaTaxYear.endYear.toString, poaTotalAmount.toCurrencyString))
+    ))
+
+    @p(id = Some("p2")) {
+        @getMessage("p2", poaTaxYear.startYear.toString, poaTaxYear.endYear.toString, (poaTaxYear.endYear + 1).toString)
```

Comment:
`poaTaxYear.endYear + 1`
you can change to improve readability 
`poaTaxYear.nextYear.endYear `

Reply:
Done.